cmd/skopeo/proxy.go

```diff
@@ -182,6 +206,30 @@ type convertedLayerInfo struct {
 	MediaType string        `json:"media_type"`
 }
 
+// mapProxyErrorCode turns an error into a known string value.
+func mapProxyErrorCode(err error) string {
+	switch {
+	case err == nil:
+		return ""
+	case errors.Is(err, syscall.EPIPE):
+		return string(proxyErrPipe)
+	case retry.IsErrorRetryable(err):
+		return proxyErrRetryable
```

Comment:
The difference here suggests that the `proxyErrorCode` type needs to be explicitly used for all of the enum values.

Reply:
Yes, thanks for that!